### ISSUE TITLE
Font is kind of hard to read

### ISSUE THREAD

**Issue by kongyang80** [mswysocki/Madison-Student-Apartment-Finder#29]
Severity: Low

The font looks fine on the iPhone and on IE9. On Chrome and Firefox, the regular sized white font looks pixelated for some reason.

**Comment by xibaca** [comment]
Humm.. interesting. It looks fine here on Chrome. What is the resolution of your screen? Did you test on all browsers at the same machine (I think so)?

Seems like we should make the font bigger so it looks nice to as many people as possible.

It is actually interesting that it works nice on the IPhone, we didn't design it with mobile devices in mind (yet).

**Comment by kongyang80** [comment]
I tried all the browser at the same time to see if they all looked similar.

The font looks the same on both my laptop and on a desktop, which both have different screen resolution.

I posted a screenshot here http://i45.tinypic.com/4lkcgh.png 
Make sure to zoom to the actual size or download the image to see how it actually looks like.

**Comment by xibaca** [comment]
Yeah, I just saw it. It is really weird. We will be using Bootstrap from Twitter, so after our big redesign of the page, that shall not be an issue anymore (as well as many other issues pointed out). =)
